### ISSUE TITLE
Why are the biases set to zero in the training?

### ISSUE THREAD

**Issue by hshustc** [phillipi/pix2pix#103]
I found that in the implementation, the biases of convolution layers in both Generator and Discriminator are set to zero before updating. I am wondering if you can tell the reason why this is happening. Thanks a lot.

refer to:
1. https://github.com/phillipi/pix2pix -> train.lua -> line 232-233 and line 262-263

**Comment by junyanz** [comment]
We set it as zero as we use `batchnorm` after each conv layer, and `batchnorm` has its own bias and gain parameters.

**Comment by appierys** [comment]
Another question: the default batch size is 1, wouldn't this hurt `batchnorm`?
